packages/kotti-ui/source/kotti-field-select/constants.ts

```diff
@@ -31,6 +31,18 @@ export const KOTTI_FIELD_SELECT_SUPPORTS: KottiField.Supports = {
 
 export const KOTTI_FIELD_SINGLE_SELECT_PROPS = {
 	...KOTTI_FIELD_SELECT_PROPS,
+	actions: {
+		default: () => null,
+		type: Array,
+		validator: (value: unknown[]): value is KottiFieldSingleSelect.Action[] =>
+			value.every(
+				(action: any): action is KottiFieldSingleSelect.Action =>
```

Comment:
any reason for choosing `any` over `unknown`

Reply:
Getting error label is not defined on type object with unknown